app/models/edition.rb

```diff
@@ -30,6 +30,9 @@ class Edition < ActiveRecord::Base
   has_many :classification_featurings, inverse_of: :edition
   has_many :links_reports, as: :link_reportable
 
+  has_many :dependencies, class_name: 'EditionDependency', foreign_key: :edition_id, dependent: :destroy
```

Comment:
now that the `foreign_key` is `edition_id` I think you should be able to omit the `foreign_key: :edition_id` declaration here, since that would be the default according to rails conventions